tmva/pymva/src/RModelParser_Keras.cxx

```diff
@@ -271,8 +271,9 @@ std::unique_ptr<ROperator> MakeKerasDense(PyObject* fLayer){
 ///
 /// For Keras's Conv layer, the names of the input tensor, output tensor, and
 /// weight tensors are extracted, along with attributes like dilation_rate,
-/// groups, kernel size, padding, strides. Padding attribute is then 
+/// groups, kernel size, padding, strides. Padding attribute is then
 /// computed for ROperator depending on Keras' attribute parameter.
+#if PY_MAJOR_VERSION >= 3   // using PyDict_GetItemWithError that is available only in Python3
```

Comment:
In Python 2.7, we have a similar function, we can maybe use that instead of throwing the runtime error.
Ref: https://docs.python.org/2.7/c-api/dict.html#c.PyDict_GetItem